### PR TITLE
Filtro para a tela de listagem de links de pagamento

### PR DESCRIPTION
**Pull request by zignd**
## Contexto
Estamos construindo a tela de listagem de links de pagamento, para isso é necessário um componente para permitir que a interação do usuário que deseja realizar uma filtragem. Essa pull request adiciona um card de filtro que será utilizado na tela de listagem de links de pagamento.

## Checklist
- Adiciona um card de filtro que será utilizado na tela de listagem de links de pagamento

## Issues linkadas
- Resolves https://app.clubhouse.io/pagar-me/story/746/listagem-de-link-criar-componente-do-searchbox-da-listagem

## Screenshots
image

### Layout:
https://www.figma.com/file/EYeEiMHAFB3N1o0g0FQHG9/Link-de-Pagamento?node-id=287%3A3704

## Como testar?
Acessar a story no storybook da pilot, encontrará em Containers > Payment Links > Filter.

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 45d468a8bd83aac8261fbb7e3f60f9b31537a598
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 45d468a8bd83aac8261fbb7e3f60f9b31537a598
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 45d468a8bd83aac8261fbb7e3f60f9b31537a598
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 45d468a8bd83aac8261fbb7e3f60f9b31537a598
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 45d468a8bd83aac8261fbb7e3f60f9b31537a598
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!  


 * Commit: 6a7244b75109ae5f3babeb3f227f0c980c881e4a
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 45d468a8bd83aac8261fbb7e3f60f9b31537a598
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!  


 * Commit: 6a7244b75109ae5f3babeb3f227f0c980c881e4a
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 45d468a8bd83aac8261fbb7e3f60f9b31537a598
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!  


 * Commit: 6a7244b75109ae5f3babeb3f227f0c980c881e4a
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 45d468a8bd83aac8261fbb7e3f60f9b31537a598
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!  


 * Commit: 6a7244b75109ae5f3babeb3f227f0c980c881e4a
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 45d468a8bd83aac8261fbb7e3f60f9b31537a598
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!  


 * Commit: 6a7244b75109ae5f3babeb3f227f0c980c881e4a
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!  


 * Commit: 0e66f7a7b62582e3c46af885eea90481fd4142ab
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 45d468a8bd83aac8261fbb7e3f60f9b31537a598
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!  


 * Commit: 6a7244b75109ae5f3babeb3f227f0c980c881e4a
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!  


 * Commit: 0e66f7a7b62582e3c46af885eea90481fd4142ab
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 45d468a8bd83aac8261fbb7e3f60f9b31537a598
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!  


 * Commit: 6a7244b75109ae5f3babeb3f227f0c980c881e4a
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!  


 * Commit: 0e66f7a7b62582e3c46af885eea90481fd4142ab
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 45d468a8bd83aac8261fbb7e3f60f9b31537a598
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!  


 * Commit: 6a7244b75109ae5f3babeb3f227f0c980c881e4a
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!  


 * Commit: 0e66f7a7b62582e3c46af885eea90481fd4142ab
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

File: packages/pilot/src/containers/PaymentLinks/Filter/index.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    setLinkNameQuery(fieldsData.linkNameQuery)
+    setActive(fieldsData.active)
+    setInactive(fieldsData.inactive)
+    onReset()
+  }
+
+  return (
+    <Grid>
+      <Row>
+        <Col palm={12} tablet={12} desk={12} tv={12}>
```
Comment (eduardogspereira) [inline packages/pilot/src/containers/PaymentLinks/Filter/index.js:42]: Acho que usando `<Flexbox>` aqui talvez já seria o suficiente, ou fica diferente?

Comment (zignd) [reply packages/pilot/src/containers/PaymentLinks/Filter/index.js]: Ajustado, aproveitei para seguir aquela sua dica de deixar a responsabilidade de ajustar o componente para o pai

**Comment by zignd** [comment]
> Na screenshot que você adicionou está faltando o ícone do calendário no campo de data.

Adicionei o ícone de calendário e ajustei o tamanho do ícone de pesquisa que estava em 16x16px mas deveria ser 12x12px.

**Pull request merged**